htdocs/stripe/class/actions_stripe.class.php

```diff
@@ -55,28 +49,24 @@ function __construct($db)
 	{
 		$this->db = $db;
 	}
-
+  
 
 	/**
-	 * formObjectOptions
 	 *
-	 * @param	array	$parameters		Parameters
-	 * @param	Object	$object			Object
-	 * @param	string	$action			Action
 	 */
 	function formObjectOptions($parameters, &$object, &$action)
 	{
 		global $db,$conf,$user,$langs,$form;
 
-		if (! empty($conf->stripe->enabled) && (empty($conf->global->STRIPE_LIVE) || GETPOST('forcesandbox','alpha')))
-		{
-			$service = 'StripeTest';
-			dol_htmloutput_mesg($langs->trans('YouAreCurrentlyInSandboxMode','Stripe'),'','warning');
-		}
-		else
-		{
-			$service = 'StripeLive';
-		}
+      if (! empty($conf->stripe->enabled) && (empty($conf->global->STRIPE_LIVE) || empty($conf->global->STRIPECONNECT_LIVE) || GETPOST('forcesandbox','alpha')))
```

Comment:
If you add this test like this, when STRIPE_LIVE is on (and STRIPECONNECT_LIVE not set, because stripe connect not used), then actions are done in test when it should be done in live (STRIPE_LIVE is on)
We should have only one var to decide if we are live or not.

Reply:
@eldy yes, because platforme can/must disabled payment/force the test mode (in case of mistake/error... ) for the subentity without change the settings of the entity. (more simple to manually change this on every entity....) when platforme return to live mode, the entity keep their original mode Live or Test.

Reply:
So if you want a second option called STRIPECONNECT_LIVE to define if we are live or not, specific to this page, you must invert the logic.
Bu default STRIPECONNECT_LIVE is not defined. And when it is not define, the STRIPE_LIVE must works as expected (LIVE or NOT).

Reply:
I have done some changes